spec/fixtures/proto-google-common-protos-0.1.9.pom

```diff
@@ -36,7 +36,6 @@
   </developers>
   <scm>
     <connection>scm:git:https://github.com/googleapis/googleapis</connection>
-    <url>${scm.url}</url>
```

Comment:
The fix in this PR broke a test case that was testing scm.url shows up in the homepage. But I believe this was an invalid pom.xml, since it's a variable that refers to itself. If I try to do something similar, mvn throws an error about a recursive element. I've corrected this invalid file, and added another test to confirm `scm.url` is getting reflected in the properties (which is what I think the intent of this was about).